infer_structcol/inference.py

```diff
@@ -126,7 +139,12 @@ def run_mcmc(data, sample, nwalkers, nsteps, theta_guess = theta_guess_default,
     theta_guess = {key:theta_guess_default[key] for key in guess_keylist}
     theta_range = {key:theta_range_default[key] for key in range_keylist}
 
-    theta = find_max_like(data, sample, theta_guess, theta_range, seed)
+    # Calculate the standard deviation of the multiple scattering calculations
+    # based on number of trajectories and number of scattering events
+    sigma = calc_sigma(theta_guess['phi'], theta_guess['radius'], theta_guess['thickness'], 
+                       sample, ntrajectories, nevents, seed=seed)
+    
+    theta = find_max_like(data, sample, theta_guess, theta_range, sigma, ntrajectories, nevents, seed)
     ndim = len(theta)
 
```

Comment:
The initial guesses already have to be pretty good to return finite log posteriors, so I think that the uncertainties won't change significantly after finding the max likelihood values. Also, the uncertainties don't seem to depend strongly on the parameters.